commcare_connect/opportunity/forms.py

```diff
@@ -418,3 +419,53 @@ def __init__(self, *args, **kwargs):
 
         choices = [(user.pk, user.username) for user in users]
         self.fields["selected_users"] = forms.MultipleChoiceField(choices=choices)
+
+
+class OpportunityVerificationFlagsConfigForm(forms.ModelForm):
+    class Meta:
+        model = OpportunityVerificationFlags
+        fields = ("duplicate", "duration", "gps", "location", "form_submission_start", "form_submission_end")
+        widgets = {
+            "form_submission_start": forms.TimeInput(attrs={"type": "time", "class": "form-control"}),
+            "form_submission_end": forms.TimeInput(attrs={"type": "time", "class": "form-control"}),
+        }
+        labels = {
+            "duplicate": "Check Duplicates",
```

Comment:
It is pretty unclear what these mean, can you add a bit more explanation so users will understand what is being checked

Reply:
Fixed. Also added some text for GPS. Please suggest changes to help texts if any. https://github.com/dimagi/commcare-connect/pull/301/commits/842fe1a3788615fb8efd1a4d01e05511e6632159